resources/views/admin/question/show.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('layouts.app')
+
+@section('content')
+
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col-md-8">
+
+      <div class="card">
+        <div class="card-header">{{ __($question->church->name) }}</div>
+
+        <div class="card-body">
+          <div class="text-md-center">{{ __($question->description) }}</div>
+          <ul class="list-group list-group-flush">
+            @foreach ($question->users as $user)
+              <li class="list-group-item">
+                <a>
+                  {{ $user->name }} - {{ $user->pivot->rating }}
```

Comment:
One pattern that might be worth exploring here is the presenter pattern. I don't know what the common practice is in Laravel, but [this article](https://freek.dev/574-simplifying-presenters-in-laravel) discusses two approaches. The goal is to hide `->pivot->` behind a presenter object, which would also enforce the Law of Demeter.

Reply:
Okay, will take a look at presenter pattern